Gems/LyShine/Code/Editor/PropertyHandlerSprite.cpp

```diff
@@ -63,7 +63,7 @@ PropertySpriteCtrl::PropertySpriteCtrl(QWidget* parent)
 
         slicerButton->setFocusPolicy(Qt::StrongFocus);
 
-        slicerButton->setIcon(QIcon("Icons/PropertyEditor/open_in.png"));
+        slicerButton->setIcon(QIcon(":/PropertyEditor/Resources/open_in.png"));

```

Comment:
Please use `:/stylesheet/img/UI20/open-in-internal-app.svg` instead, which looks the same and is already used in the other property asset controls.

Reply:
Using the pre-existing icon will also take care of the `Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/Resources/Icons.qrc` conflict since you won't need to touch that file anymore

Reply:
done.